base_layer/core/Cargo.toml

```diff
@@ -63,6 +63,7 @@ croaring = { version = "=0.3.9", optional = true }
 config = { version = "0.9.3" }
 strum = "0.17.1"
 strum_macros = "0.17.1"
+async-std = "1.5.0"
```

Comment:
We already have a runtime. async-std is a largely non-compatible "competitor" to tokio

Reply:
async-std was removed.